### ISSUE TITLE
Please allow override of "adjusted prevalence"

### ISSUE THREAD

**Issue by PatrickDRusk** [microcovid/microcovid#179]
**TL;DR:** Please allow an override of the "adjusted prevalence" percentage in Step 1

From a bit of experimentation, it appears that your adjusted prevalence calculation is based on the completely logical idea that the lower the test positivity, the higher the percentage of true infections the county is successfully finding. I believed that myself for quite some time, but have become convinced that it is not true. Massachusetts has very low test positivity, but we are most likely finding no better than 1/7th of our cases, probably more like 1/9th. Either that, or we are doing *much worse* than other states at keeping our infected alive.

You could test this yourself by taking the prevalence that your algorithm suggests, and calculating using that prevalence what the IFR would be for the counties/states. You will find in doing so that your IFRs will be wildly different from county-to-county, or state-to-state, although there is no good reason to believe that to be true at the state level.

If the test positivity hypothesis was correct, you would find that test positivity would highly correlate with CFR (the fatality rate of confirmed cases). But it doesn't. That was one of the most surprising findings in the data for me.

I believe the reason for that is that states with very low positivity, like my home state of MA, do a lot of surveillance testing of asymptomatic people. That sort of testing would have very low positivity largely equivalent to the prevalence. But states getting hit hard generally test diagnostically, testing symptomatic people and their close contacts. That sort of testing has a much higher positivity rate. Unfortunately, every state blends together their surveillance and diagnostic testing numbers, so that you can't piece out how thorough their diagnostic testing is.

As a result, I would suggest that you make the "adjusted prevalence" editable, with an initial suggested value based on your calculation.
